builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,207 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
```

Comment:
I don't think we need to add the user to `release/1.2` as the branch is deprecated. However, we should introduce it to `main` as this is a pro-active security fix. The default user for docker is 1000 and it can collide with a different user there. The new user is allocated for edgeagentuser to prevent the user collision.

Reply:
made change